Types error for python 3.13 in CI

### What happened?

Currently python 3.13 raises the following in CI (https://github.com/vega/altair/actions/runs/17256395317/job/48990491950):

```cmd
tests/vegalite/v6/test_theme.py:1073: error: Unused "type: ignore" comment 
[unused-ignore]
            ConfigKwds.__readonly_keys__,  # type: ignore[attr-defined]
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
tests/vegalite/v6/test_theme.py:1074: error: Unused "type: ignore" comment 
[unused-ignore]
            ConfigKwds.__mutable_keys__,  # type: ignore[attr-defined]
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Found 2 errors in 1 file (checked 395 source files)
Error: Process completed with exit code 1.
```

### What would you like to happen instead?

_No response_

### Which version of Altair are you using?

_No response_

--- a/tests/vegalite/v6/test_theme.py
+++ b/tests/vegalite/v6/test_theme.py
@@ -1070,8 +1070,9 @@ def test_theme_config(theme_func: Callable[[], ThemeConfig], chart) -> None:
 def config_keys() -> frozenset[str]:
     return ConfigKwds.__required_keys__.union(
         ConfigKwds.__optional_keys__,
-        ConfigKwds.__readonly_keys__,  # type: ignore[attr-defined]
-        ConfigKwds.__mutable_keys__,  # type: ignore[attr-defined]
+        # TODO: Remove [attr-defined,unused-ignore] when mypy properly supports TypedDict attributes
+        ConfigKwds.__readonly_keys__,  # type: ignore[attr-defined,unused-ignore]
+        ConfigKwds.__mutable_keys__,  # type: ignore[attr-defined,unused-ignore]
     )
 
 

